4.sc_Morphology_Neural_Network_MLP_Model/0.model_training/scripts/Hyperparameter_Optimization_model_regression.py

```diff
@@ -309,7 +335,10 @@
 else:
     pass
 
-# %%
+
+# In[22]:
+
+
 # Call visualization function
 # calling the testing function and outputing list values of tested model
 if params.MODEL_TYPE == "Multi_Class" or params.MODEL_TYPE == "Regression":
```

Comment:
```suggestion
if any(params.MODEL_TYPE == model_type for model_type in ["Multi_Class", "Regression"]):
```

reduce code duplication